zips/zip-0253.md

```diff
@@ -31,10 +31,10 @@ This proposal defines the deployment of the NU6 network upgrade.
 The primary sources of information about NU6 consensus protocol changes are:
 
 * The Zcash Protocol Specification [^protocol].
-* Network Upgrade Mechanism [^zip-0200].
-* Lockbox Funding Streams [^zip-2001].
-* Block Reward Allocation for Non-Direct Development Funding [^draft-nuttycom-funding-allocation].
-* Blocks should balance exactly [^zip-0236].
+* ZIP 200: Network Upgrade Mechanism [^zip-0200].
```

Comment:
* ZIP 207: Funding Streams [^zip-0207].

Reply:
NACK, we already mention ZIP 207 below as being updated (by ZIP 2001 which this section already references).